mne/source_estimate.py

```diff
@@ -1791,6 +1808,89 @@ def plot(self, src, subject=None, subjects_dir=None, mode='stat_map',
             initial_time=initial_time, initial_pos=initial_pos,
             verbose=verbose)
 
+    # Override here to provide the volume-specific options
+    @verbose
+    def extract_label_time_course(self, labels, src, mode='auto',
+                                  allow_empty=False, trans=None,
+                                  mri_resolution=True, verbose=None):
+        """Extract label time courses for lists of labels.
+
+        This function will extract one time course for each label. The way the
+        time courses are extracted depends on the mode parameter.
+
+        Parameters
+        ----------
+        %(eltc_labels)s
+        %(eltc_src)s
+        %(eltc_mode)s
+        %(eltc_allow_empty)s
+        %(eltc_trans)s
+        %(eltc_mri_resolution)s
+        %(verbose_meth)s
+
+        Returns
+        -------
+        %(eltc_returns)s
+
+        See Also
+        --------
+        extract_label_time_course : Extract time courses for multiple STCs.
+
+        Notes
+        -----
+        %(eltc_mode_notes)s
+        """
+        return extract_label_time_course(
+            self, labels, src, mode=mode, return_generator=False,
+            allow_empty=allow_empty, verbose=verbose)
```

Comment:
you don't pass `trans` or `mri_resolution` parameters here. I am confused

Reply:
Just a bug, need to write tests to use this method in addition to the function !

Reply:
Fixed